core/modules/wildcard_match.cc

```diff
@@ -335,7 +335,10 @@ int WildcardMatch::DelEntry(struct WmTuple *tuple, wm_hkey_t *key) {
     tuple->ht.Close();
 
     num_tuples_--;
-    memmove(&tuples_[idx], &tuples_[idx + 1],
+
+    // shift remaing tuples to the left
+    memmove(reinterpret_cast<void*>(&tuples_[idx]),
```

Comment:
At some point we should rework /util/histogram.h, for the following desired properties:
* it should not internally impose any specific time units (e.g., nanoseconds or cycles).
  * actually it doesn't need to be 'time' in the first place.
* logarithmic binning, since latency tends to be long-tailed.
* not really required at the moment, but ideally it can be templatized to support floating point numbers as well.